Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidAssignmentInConditionAnalyzer.cs

```diff
@@ -37,8 +38,7 @@ public override void Initialize(AnalysisContext context)
 
 		private void AnalyzeIfStatement(SyntaxNodeAnalysisContext context)
 		{
-			GeneratedCodeDetector generatedCodeDetector = new();
-			if (generatedCodeDetector.IsGeneratedCode(context))
```

Comment:
Was moving the constructor a noticeable improvement? (I do think we can safely remove most of the GeneratedCodeDetector logic, as it's redundant with ConfigurationFlags.None.)

Reply:
It seems like, but the measurement are unstable, see issue #402